_includes/codeTable.html

```diff
@@ -207,7 +207,7 @@
         {%- for admin in govLevels[1] -%}
           {%- for release in admin[1].releases -%}
           {%- unless release.tags.en contains site.excludeTag -%}
-          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 349px;">
+          <tr class="col-xs-12 col-sm-6 col-md-4" style="vertical-align: top; min-height: 350px;">
```

Comment:
Any idea why the equal height isn't working?

Looking at the Style Guide, I see you don't need the style attribute here.  WET will add it with the min-height, but not the right one.  Even if I remove it it still does'nt work

Reply:
I think the equal height doesn't work due to where the box is created - I'll take a closer look

Also can double check the inline styling to remove it where it's not needed.

Reply:
Opened issue to be resolved in new PR